meta-rinato/conf/machine/rinato.conf

```diff
@@ -0,0 +1,41 @@
+
+#@TYPE: Machine
+#@NAME: rinato 
+#@DESCRIPTION: Machine configuration for the Gear 2 / Gear 2 Neo and maybe some related watches, running mainline kernel.
+
+require conf/machine/include/arm/armv7a/tune-cortexa7.inc
+DEFAULTTUNE = "armv7vehf-neon"
+
+MACHINE_FEATURES = "alsa bluetooth usbgadget ext2"
+
+# Serial console via USB serial gadget
+SERIAL_CONSOLE = "115200 ttyGS0"
+
+IMAGE_FSTYPES:append = " ext4 "
+IMAGE_ROOTFS_ALIGNMENT = "4"
+
+PREFERRED_PROVIDER_virtual/mesa ?= "mesa"
+PREFERRED_PROVIDER_virtual/egl ?= "mesa"
+PREFERRED_PROVIDER_virtual/libgl ?= "mesa"
+PREFERRED_PROVIDER_virtual/libgbm ?= "mesa"
+PREFERRED_PROVIDER_virtual/gles ?= "mesa"
+PREFERRED_PROVIDER_virtual/libgles1 ?= "mesa"
+PREFERRED_PROVIDER_virtual/libgles2 ?= "mesa"
+
+MACHINE_DISPLAY_ROUND = "false"
```

Comment:
You don't need to explicitly set these to 'false' since they all default to false anyway.

No need to fix imo. For the speaker and microphone related variables I agree that mentioning them explicitly makes sense.

Reply:
I'd prefer to be explicit here, if you don't mind too much.

Reply:
Yeah no need :D